README.md

```diff
@@ -205,6 +205,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Auth0](https://auth0.com) | Easy to implement, adaptable authentication and authorization platform | `apiKey` | Yes | Yes |
 | [Micro User Service](https://m3o.com/user) | User management and authentication | `apiKey` | Yes | No |
+| [GetOTP](https://otp.dev/en/docs/) | Implement OTP flow quickly  | `apiKey` | Yes | No |
```

Comment:
Please arrange it in the alphabetical order

Reply:
Done